graphql-java-support/src/main/java/com/apollographql/federation/graphqljava/tracing/FederatedTracingInstrumentation.java

```diff
@@ -37,14 +37,16 @@
 import java.util.concurrent.locks.ReadWriteLock;
 import java.util.concurrent.locks.ReentrantReadWriteLock;
 import java.util.function.Consumer;
+import java.util.function.Function;
 
 import static graphql.execution.instrumentation.SimpleInstrumentationContext.whenCompleted;
 import static graphql.schema.GraphQLTypeUtil.simplePrint;
 
 public class FederatedTracingInstrumentation extends SimpleInstrumentation {
+    public static final String HEADER_NAME = "apollo-federation-include-trace";
+    public static final String HEADER_VALUE = "ftv1";
```

Comment:
Made public so integrators who provide their own evaluateContextFunction can leverage the same header name and value in their implementation.

Reply:
May want to consider naming this a bit more descriptive if opening up for public use. An alternative would be to provide static accessor methods for the header name and value so that consumers lock onto that rather than a const.